committee-steering/sig-governance.md

```diff
@@ -0,0 +1,465 @@
+# SIG Governance Template
+
+## Goals
+
+The following document outlines a template governance structure for
+SIGs to follow.  Its goals are to provide the necessary structure
+for SIGs to self organize and self govern.
+The focus of this document is on the following:
+
+- outline organizational responsibilities each SIG must fulfill
+  - internally facing to the SIG
+  - externally facing to the project and community
+- outline organizational roles and what they are responsible for
+- outline processes and tools to facilitate roles
+
+The focus of this template is providing structure to the following areas:
+
+- The role of technical leadership and how technical leaders are selected
+- The role of administration and how administrators are selected
+- How technical decisions are proposed and made within a SIG
+- How to ensure continued health of tests
+
+## How to use it
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+## FAQ
```

Comment:
Just a thought: It'd be nice to have the FAQ section look like this:
```
FAQ
- [Why is it so long?]<link>
- [Why so much structure]<link>
- [Why use mailing lists for tracking roles]<link>
```
Just to help us get to the meat quicker. A person may be inclined to read the whole FAQ first and I don't know if that should be the goal.